src/components/Common/ButtonWithArrow.css

```diff
@@ -0,0 +1,35 @@
+.ButtonWithArrow {
+  display: flex !important;
+  align-items: baseline;
+  gap: 6px;
+  font-size: 11px !important;
+  line-height: 18px !important;
+}
+
+.ui.button.basic.ButtonWithArrow {
+  color: var(--black-600) !important;
+  font-weight: var(--weight-semi-bold) !important;
+}
+
+@media (max-width: 350px) {
```

Comment:
I'd change this to `min-width: 351px` and invert the properties here. Avoid using max-width for a mobile-first approach as it is easier to read.